ci/tasks/send-job-release-info-to-hosted-graphite.yml

```diff
@@ -0,0 +1,20 @@
+---
+platform: linux
+image_resource:
+  type: registry-image
+  source:
+    repository: alpine
+    tag: 3.16
+params:
+  ENVIRONMENT:
+  APP_NAME:
+  RELEASE_NUMBER:
+  HOSTED_GRAPHITE_API_TOKEN:
+run:
+  path: sh
+  args:
+    - -ec
+    - |
+      RELEASE=$(echo "${RELEASE_NUMBER}" | cut -d'-' -f 1)
+      METRIC_DATA="${HOSTED_GRAPHITE_API_TOKEN}.ci.concourse.job.${ENVIRONMENT}.${APP_NAME}.releaseNumber $RELEASE"
```

Comment:
We'll want to include the pipeline and job into the metric name, instead of `ENVIRONMENT` and `APP_NAME`. These should be available from the [list of built-in environment variables](https://concourse-ci.org/implementing-resource-types.html#resource-metadata) that Concourse provides.

We also need to call this task from the pipeline file above (and make sure that either `RELEASE_NUMBER` is provided, or we change the shell script here to use something else for our numeric value).